plugins/colordialog/plugin.js

```diff
@@ -38,13 +39,22 @@ CKEDITOR.plugins.colordialog = {
 				}
 				callback.call( scope, color );
 			};
+			onShow = function( evt ) {
+				if ( options && options.selectionColor ) {
+					evt.data.colordialog = evt.data.colordialog || {};
```

Comment:
I don't see a need to add additional namespace. I'd pass all the data directly into `evt.data`.